va/proto/va.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto2";
+
+package va;
+option go_package = "proto";
+
+import "core/proto/core.proto";
+
+service VA {
+	rpc IsSafeDomain(IsSafeDomainRequest) returns (IsDomainSafe) {}
+	rpc PerformValidation(PerformValidationRequest) returns (ValidationResult) {}
+}
+
+message IsSafeDomainRequest {
+	optional string domain = 1;
+}
+
+message IsDomainSafe {
+	optional bool isSafe = 1;
```

Comment:
I previously gave the advice that all protobuf fields should be optional, as a general rule. I think that may have been bad advice here, sorry. Clearly this protobuf only makes sense if this field is filled, and having a direct value for this bool rather than a pointer will make a lot of this code clearer. I think, weakly, IsSafeDomainRequest has the same property. I'd recommend changing these to:

```
+message IsSafeDomainRequest {
 +	required string domain = 1;
 +}
 +
 +message IsDomainSafe {
 +	required bool isSafe = 1;
```